packages/truffle-artifactor/utils.js

```diff
@@ -9,4 +10,22 @@ const writeArtifact = (_completeArtifact, _outputPath) => {
   );
 };
 
-module.exports = { writeArtifact };
+const finalizeArtifact = (
+  _normalizedExistingArtifact,
+  _normalizedNewArtifact
```

Comment:
What's up with all these underscores in this file

Reply:
Probably not necessary now that the helpers are in a separate file and not really "private" anymore.
If it's confusing and/or inconsistent, I can remove 'em.